source/operations/cert-manager/cert-manager-tenants.rst

```diff
@@ -0,0 +1,272 @@
+      - Replace ``myminio-tls`` with the name of your secret, if different.
+
+4) Deploy the tenant using cert-manager for TLS certificate management
+----------------------------------------------------------------------
+
+When deploying a Tenant, you must set the TLS configuration such that:
+
+- The Tenant does not automatically generate its own certificates (``spec.requestAutoCert: false``) *and*
+- The Tenant has a valid cert-manager reference (``spec.externalCertSecret``)
+
+This directs the Operator to deploy the Tenant using the cert-manager certificates exclusively.
+
+The following YAML ``spec`` provides a baseline configuration meeting these requirements:
+
+.. code-block:: yaml
+   :emphasize-lines: 6,9,11
+
+   apiVersion: minio.min.io/v2
+   kind: Tenant
+   metadata:
+     name: myminio
+     namespace: tenant-1
+   spec:
+   ...
+     ## Disable default tls certificates.
+     requestAutoCert: false
+     ## Use certificates generated by cert-manager.
+     externalCertSecret:
+       - name: myminio-tls
+         type: cert-manager.io/v1
+   ...
+
+5) Trust the tenant's CA in MinIO Operator
+------------------------------------------
+
+The MinIO Operator does not trust the tenant's CA by default.
+To trust the tenant's CA, you must pass the certificate to the Operator as a secret.
+
+To do this, create a secret with the prefix ``operator-ca-tls-`` followed by a unique identifier in the `minio-operator` namespace.
+
+MinIO Operator mounts and trusts **all** certificates issued by the provided Certificate Authorities. 
+This is required because the MinIO Operator performs health checks using the ``/minio/health/cluster`` endpoint.
+
+Create ``operator-ca-tls-tenant-1`` secret
+++++++++++++++++++++++++++++++++++++++++++
+
+Copy the tenant's cert-manager generated CA public key (``ca.crt``) into the `minio-operator` namespace. 
+This allows Operator to trust the cert-manager issued CA and all certificates derived from it.
+
+1. Create a ``ca.crt`` file containing the CA:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl get secrets -n tenant-1 tenant-1-ca-tls -o=jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+
+2. Create the secret:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl create secret generic operator-ca-tls-tenant-1 --from-file=ca.crt -n minio-operator
+
+.. tip::
+
+   In this example we chose a secret name of ``operator-ca-tls-tenant-1``. 
+   We used the tenant namespace ``tenant-1`` as a suffix for easy identification of which namespace the CA comes from.
+   Use the name of your tenant namespace for easier linking secrets to the related resources.
+
+6) Deploy the tenant 
+--------------------
+
+With the Certificate Authority and ``Issuer`` in place for the tenant's namespace, you can now :ref:`deploy the object store tenant <minio-k8s-deploy-minio-tenant>`.
+
+Use the tenant YAML modified above to disable AutoCert and reference the secret you generated.
```

Comment:
```suggestion
Use the modified baseline tenant YAML to disable AutoCert and reference the secret you generated.
```
directional text ... (yeah, adjective stack instead but trying to get through ...)